play-dl/YouTube/classes/LiveStream.ts

```diff
@@ -117,7 +117,7 @@ export class Stream {
     private per_sec_bytes : number;
     private content_length : number;
     private video_url : string;
-    private timer : NodeJS.Timer;
+    private timer : NodeJS.Timer | null;
```

Comment:
Here, add the null type.

Reply:
Adding null type is not required. Since I will be creating a Timer at Class initiation.